as.c

```diff
@@ -3,6 +3,7 @@
 #include <string.h>
 
 #include "vm.h"
+#include "opcode.h"
```

Comment:
Should this move the instruction too? I think the instruction can also be generated on-the-fly too, if we use a customized script to generate `opcode.h`